publication/ver11/societal_impact.md

```diff
@@ -0,0 +1,216 @@
+for automation of processes (for example in a factory), for monitoring (for example,
+to detect environmental conditions), or for coordinating multiple devices to 
+provide a new service (for example, by combining and analysing data from sensors to
+detect intrusions and implement a home security system).  IoT devices can also
+be useful to combine other devices and data sources in useful ways, for example
+to physically display data in real-time (for example, to provide a weather alert)
+or to provide an alternative physical interface to some digital functionality
+(which provide opportunities to improve accessibilty with custom interfaces).
+
+Specifically, the WoT standards provide consistent metadata for the access of 
+IoT systems, so that IoT devices following different standards (or no standards...) can
+interoperate.  Therefore, it specifically targets the case where multiple IoT devices
+from different manufacturers and possibly different standards ecosystems need to be used 
+together.
+
+The WoT standards can be considered "essential" when this metadata is not available
+by other means or would be expensive or inconvenient to obtain, or where machine-readable
+metadata is needed to automate integration.
+
+When combined with a browser, the consistent metadata provided by WoT standards
+can be used to automatically build web user interfaces for IoT devices, making them
+more consistently accessible.  This may or may not be done by directly talking to 
+IoT devices.  Very often an IoT system will digest the metadata about a set of
+IoT devices and use this to generate a consistent interface, such as a dashboard.
+
+Some usages of WoT may be safety-critical, for example, an IoT interface to a factory
+actuator or to a fall-detection system in the home.  A government may also use WoT
+to provide access to public services, including critical mechanisms such as access
+to environmental monitoring systems (which might in turn be used to warn vulnerable
+people about air pollution states they need to be wary of). 
+
+The WoT standards primarily describe metadata and how to access it.  To mitigate
+failures, such as the inability to access metadata if a WoT Discovery service
+is unavailable, such systems should use WoT Discovery and metadata (WoT Thing
+Descriptions) for initial connections and if updated metadata is not available,
+use the last known state of this metadata.
+
+### 2. [What kinds of activities could your specification become a part of that you are not designing for?](https://w3ctag.github.io/societal-impact-questionnaire/#unexpected-use)
+WoT does not directly address accessibility but IoT systems generally can be
+used to build custom user interfaces, including those that make systems easier
+to use (e.g. large buttons to replace difficult to use controls) or map sensory
+modalities (e.g. mapping visual indicators to audible ones or vice-versa).
+Unfortunately, the current WoT specification describes only network behavior,
```

Comment:
I would replace "network" with "communication" or "interaction"

Reply:
communication is more general and could be interpreted in this context to mean human communication, like speech or display information.  As a compromise, changed "network behavior" to "network communication and interactions".